sale_invoice_blocking/views/sale_order.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <record id="view_order_form_invoice_blocking_reason" model="ir.ui.view">
+        <field name="model">sale.order</field>
+        <field name="inherit_id" ref="sale.view_order_form" />
+        <field name="arch" type="xml">
+            <field name="client_order_ref" position="after">
+                <field name="invoice_blocking_reason_id" />
+            </field>
+            <xpath
+                expr="//button[@name='%(sale.action_view_sale_advance_payment_inv)d'][contains(concat(' ',normalize-space(@class),' '),' btn-primary ')]"
+                position="attributes"
+            >
+                <attribute
```

Comment:
@damdam-s Why not modifying invoice_status value computation to 'no' if invoice_blocking_reason_id is set that avoid doing some attrs modifications in view ?

Reply:
Thank you @rousseldenis for the suggestion. It's a very good idea indeed.